src/__screenshot_tests__/carousel-screenshot-test.tsx

```diff
@@ -17,6 +17,16 @@ test('Carousel mobile', async () => {
     expect(await page.screenshot()).toMatchImageSnapshot();
 });
 
+test('Carousel mobile selecting second page initially', async () => {
```

Comment:
I'd rename the test to
```suggestion
test('Carousel mobile with initialActiveItem=1', async () => {
```
Because we can select the active item, not the page.
Same for the other tests